shell/packages/sandstorm-ui-grainlist/grainlist.html

```diff
@@ -4,7 +4,7 @@
   <div class="grain-list">
     <div class="top-row">
     <div class="non-usage-info">
-    <h1>Grains</h1>
+    <h1>Grains <a href="#" class="question-mark"></a></h1>
```

Comment:
Can this be a button rather than an `<a>`?  I more-or-less believe than any `<a href="#">` is semantically wrong.